client/src/components/App.jsx

```diff
@@ -3,17 +3,29 @@ import RecentlyViewed from './RecentlyViewed.jsx';
 import Header from './Header.jsx';
 import Footer from './Footer.jsx';
 
-import dummyData from '../../../dummyData.js';
+import getRecentItems from '../lib/getAjax.js';
 
 class App extends React.Component {
   constructor(props) {
     super(props);
 
     this.state = {
-      items: dummyData // array of objects
+      items: [] // array of objects
     }
   }
 
+  componentDidMount() {
+    getRecentItems((err, data) => {
```

Comment:
Although the AJAX request works, I believe that defining the request in componentDidMount directly will eliminate the need to export and import the AJAX request file. As a result, you will reduce the lines of code needed to be written.

Reply:
Good point! I refactored the code based off your feedback. Thank you!